Assets/Project/Modules/Enemies/Slime/Scripts/SlimeRefactor/SlimeMovement.cs

```diff
@@ -0,0 +1,70 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.AI;
+using Random=UnityEngine.Random;
+
+public class SlimeMovement : ISlimeComponent
```

Comment:
Rename this to something for all enemies ("NavMeshMovement")

Reply:
I think the movement may become too specific for each enemy. This one is intended to be the simplest enemy and already has some specific methods, such as the ApplyExplosionForce method, to apply forces when divided. And in this sprint, it will also have a behavior to avoid the closest slimes.